.lintr

```diff
@@ -7,4 +7,4 @@ linters: with_defaults(
   single_quotes_linter = NULL,
   trailing_blank_lines_linter = NULL
   )
-exclusions: list("R/hexamers.R", "inst/doc/intro.R")
+exclusions: list("R/hexamers.R", "inst/doc/intro.html")
```

Comment:
"inst/doc/intro.R" does not exist – the intro file is an html file. 
I added intro.html to the exclusions since I assumed it _shouldn't_ be checking
the html file (and I assumed that exclusions was a list of files lintr shouldn't check)